pypeit/spectrographs/spectrograph.py

```diff
@@ -143,6 +143,9 @@ def load_raw_frame(self, raw_file, det=None):
             img = img.T
         elif self.detector[_det - 1]['dispaxis'] == -1:
             img = np.flip(img.T, axis=0)
+        elif self.detector[_det - 1]['dispaxis'] == 2:
```

Comment:
My original suggestion was that we add a keyword to DetectorPar, something like `dispflip`.  Then the code would look like this:

```
if self.detector[_det-1]['dispaxis'] == 1:
    # Transpose so that wavelength is along rows
    img = img.T
if self.detector[_det-1]['dispflip']:
    # Flip so that pixel number correlates with wavelength
    img = np.flip(img, axis=0)
```

Reply:
I have put back the deleted line in KeckTelescopePar

Reply:
I've added this now Kyle. I think we are good.